<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>872</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MIPS汇编语言动态调试器</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/logo.png</normaloff>:/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>261</width>
      <height>451</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>561</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="registers">
     <item row="10" column="3">
      <widget class="QLabel" name="reg_name3_6">
       <property name="text">
        <string>hi</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QLabel" name="reg_name2_4">
       <property name="text">
        <string>$s4</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="10" column="4">
      <widget class="QLabel" name="reg_name4_6">
       <property name="text">
        <string>lo</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="reg_name1">
       <property name="text">
        <string>$at</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="reg_name4">
       <property name="text">
        <string>$a0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="reg3"/>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="reg_name3_2">
       <property name="text">
        <string>$t1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QLineEdit" name="reg21"/>
     </item>
     <item row="1" column="4">
      <widget class="QLineEdit" name="reg4"/>
     </item>
     <item row="5" column="5">
      <widget class="QLineEdit" name="reg17"/>
     </item>
     <item row="7" column="0">
      <widget class="QLineEdit" name="reg18"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="reg_name2">
       <property name="text">
        <string>$v0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="8" column="4">
      <widget class="QLabel" name="reg_name4_5">
       <property name="text">
        <string>$gp</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QLabel" name="reg_name3_4">
       <property name="text">
        <string>$s5</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLabel" name="reg_name1_6">
       <property name="text">
        <string>$ra</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="reg_name0_3">
       <property name="text">
        <string>$t4</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="reg_name0_2">
       <property name="text">
        <string>$a2</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="10" column="5">
      <widget class="QLabel" name="reg_name5_6">
       <property name="text">
        <string>raw Hex</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="reg_name1_3">
       <property name="text">
        <string>$t5</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="5">
      <widget class="QLabel" name="reg_name5_4">
       <property name="text">
        <string>$s7</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="reg_name1_5">
       <property name="text">
        <string>$t9</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="reg_name0_6">
       <property name="text">
        <string>$s8</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="9" column="5">
      <widget class="QLineEdit" name="reg29"/>
     </item>
     <item row="11" column="2">
      <widget class="QLineEdit" name="reg32"/>
     </item>
     <item row="9" column="0">
      <widget class="QLineEdit" name="reg24"/>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="reg_name4_2">
       <property name="text">
        <string>$t2</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLineEdit" name="reg9"/>
     </item>
     <item row="10" column="2">
      <widget class="QLabel" name="reg_name2_6">
       <property name="text">
        <string>pc</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLineEdit" name="reg5"/>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="reg_name2_3">
       <property name="text">
        <string>$t6</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="reg_name0_4">
       <property name="text">
        <string>$s2</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLabel" name="reg_name3_3">
       <property name="text">
        <string>$t7</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="reg_name2_2">
       <property name="text">
        <string>$t0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="11" column="4">
      <widget class="QLineEdit" name="reg34"/>
     </item>
     <item row="5" column="3">
      <widget class="QLineEdit" name="reg15"/>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="reg_name1_2">
       <property name="text">
        <string>$a3</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="9" column="2">
      <widget class="QLineEdit" name="reg26"/>
     </item>
     <item row="11" column="3">
      <widget class="QLineEdit" name="reg33"/>
     </item>
     <item row="7" column="4">
      <widget class="QLineEdit" name="reg22"/>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="reg_name1_4">
       <property name="text">
        <string>$s3</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="9" column="3">
      <widget class="QLineEdit" name="reg27"/>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="reg19"/>
     </item>
     <item row="11" column="0">
      <widget class="QLineEdit" name="reg30"/>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="reg_name5">
       <property name="text">
        <string>$a1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLineEdit" name="reg31"/>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="reg25"/>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="reg_name3">
       <property name="text">
        <string>$v1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QLineEdit" name="reg10"/>
     </item>
     <item row="6" column="4">
      <widget class="QLabel" name="reg_name4_4">
       <property name="text">
        <string>$s6</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QLabel" name="reg_name5_3">
       <property name="text">
        <string>$s1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="8" column="5">
      <widget class="QLabel" name="reg_name5_5">
       <property name="text">
        <string>$sp</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QLabel" name="reg_name2_5">
       <property name="text">
        <string>$k0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QLineEdit" name="reg16"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="reg_name0_5">
       <property name="text">
        <string>$t8</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="reg6"/>
     </item>
     <item row="11" column="5">
      <widget class="QLineEdit" name="reg35"/>
     </item>
     <item row="4" column="4">
      <widget class="QLabel" name="reg_name4_3">
       <property name="text">
        <string>$s0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="reg8"/>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="reg13"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="reg1"/>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="reg2"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="reg7"/>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="reg0"/>
     </item>
     <item row="7" column="5">
      <widget class="QLineEdit" name="reg23"/>
     </item>
     <item row="7" column="2">
      <widget class="QLineEdit" name="reg20"/>
     </item>
     <item row="5" column="0">
      <widget class="QLineEdit" name="reg12"/>
     </item>
     <item row="5" column="2">
      <widget class="QLineEdit" name="reg14"/>
     </item>
     <item row="8" column="3">
      <widget class="QLabel" name="reg_name3_5">
       <property name="text">
        <string>$k1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="9" column="4">
      <widget class="QLineEdit" name="reg28"/>
     </item>
     <item row="3" column="5">
      <widget class="QLineEdit" name="reg11"/>
     </item>
     <item row="2" column="5">
      <widget class="QLabel" name="reg_name5_2">
       <property name="text">
        <string>$t3</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="reg_name0">
       <property name="text">
        <string>$zero</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>400</y>
      <width>561</width>
      <height>35</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>开始</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>200</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>单步</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>200</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>重置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="showRamButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>200</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>显示RAM</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>872</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpenRom"/>
    <addaction name="actionOpenRam"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actionfhh"/>
    <addaction name="separator"/>
    <addaction name="actionQt"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionfhh">
   <property name="text">
    <string>软件……</string>
   </property>
  </action>
  <action name="actionQt">
   <property name="text">
    <string>Qt……</string>
   </property>
  </action>
  <action name="actionOpenRom">
   <property name="text">
    <string>打开ROM……</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionOpenRam">
   <property name="text">
    <string>打开RAM……</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
